src/client/components/terminal/index.ts

```diff
@@ -305,6 +305,12 @@ export class TerminalView extends LitElement {
     #terminal {
       position: relative;
     }
+
+    /** xf styles */
+    .xterm-bg-257,
+    .xterm-bg-7 {
+      background-color: var(--vscode-editor-selectionBackground) !important;
```

Comment:
@degrammer will we have to worry about any unintended side-effects introducing these two color mappings?